src/index.js

```diff
@@ -1,15 +1,24 @@
-
-
 import './index.css';
-
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {BrowserRouter, Route} from "react-router-dom"
+import './index.css';
```

Comment:
Consider following the already mentioned import guidelines.
Group the imports by purpose. First go third party imports, last - css.